docs/spec_usdc.rst

```diff
@@ -0,0 +1,760 @@
+MapperArg             5    **(Internal to Pixar)** Arguments for a mapper
+Prim                  6    A Prim specifier
+PseudoRoot            7    The Pseudo Root that exists for all Sdf Layers
+Relationship          8    A relationship description
+RelationshipTarget    9    The Relationship target
+Variant               10   A definition of a specific variant
+VariantSet            11   A group of variants]
+NumSpecTypes          12   A sentinel to represent the number of types
+==================   ===  ========================
+
+Constructing the Layer
+======================
+
+Once you have read the Preamble, you can then start constructing the prim hierarchy of this
+individual USD Crate file that can act as a layer within a USD composition.
+
+.. warning::
+
+    As noted above, implementation of this document only allows for reading a single USD
+    file layer. It does not describe composition however as that is a higher level functionality
+    of the USD library.
+
+You should now have a mapping of:
+
+* Prim and Property Paths from the Paths section
+* Fields consisting of their associated Token and Value Representation.
+* Field Sets consisting of groups of Fields
+* Specs that associate paths with Field Sets and a Spec Type
+
+Therefore each node consists of:
+
+* A Path
+* A SpecType
+* A Field Set
+
+Fields are made up of Value Representations that are deferred for parsing.
+See the section below on how to parse the range of Value Representations.
+
+Layer Metadata
+--------------
+
+Layer Metadata is derived from the Field Sets on the PseudoRoot.
+
+Common metadata in layers are, but not limited to:
+
++--------------------+------------+----------------------------------------------------+
+| Name               | Type       | Description                                        |
++====================+============+====================================================+
+| comment            | string     | Top level comment for this layer                   |
++--------------------+------------+----------------------------------------------------+
+| customLayerData    | dictionary | Custom user specified data                         |
++--------------------+------------+----------------------------------------------------+
+| defaultPrim        | token      | The prim to use when this layer is referenced      |
++--------------------+------------+----------------------------------------------------+
+| documentation      | string     | Top level documentation for this layer             |
++--------------------+------------+----------------------------------------------------+
+| metersPerUnit      | double     | The scale unit for this layer                      |
++--------------------+------------+----------------------------------------------------+
+| primChildren       | token[]    | A list of top level children to limit the layer to |
```

Comment:
`primChildren` and `properties` in Layer and Prim meta uses TokenVector(41), not array of `token`

```
name = "primChildren", value = ty: 41, isArray: 0, isInlined: 0, isCompressed: 0 
```